src/ui/hexedit.cc

```diff
@@ -231,6 +349,13 @@ QModelIndex HexEdit::selectedChunk() {
   return chunkSelectionModel_->currentIndex();
 }
 
+void HexEdit::createAction(util::settings::shortcuts::ShortcutType type, const std::function<void ()>& f) {
+  auto action = ShortcutsModel::getShortcutsModel()->createQAction(
+        type, this, Qt::WidgetWithChildrenShortcut);
```

Comment:
Seems I missed this in previous PRs ;) Should be 4 spaces, see https://google.github.io/styleguide/cppguide.html#Function_Calls